### PR TITLE
Create _github-pages-challenge-MrPop-rookie.snbhowmik.json

### PR DESCRIPTION
**Pull request by snbhowmik**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

image

### CODE REVIEW

File: domains/_github-pages-challenge-MrPop-rookie.snbhowmik.json
Comment (CuteDog5695) [inline domains/_github-pages-challenge-MrPop-rookie.snbhowmik.json:1]: snbhowmik.is-a.dev doesn't exist, so you can't make a nested domain on it

File: domains/_github-pages-challenge-MrPop-rookie.snbhowmik.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  }
+  "record": {
+    "CNAME": "MrPop-rookie.github.io",
+    "TXT": "9a4944b6a34448f1edc9f4079db605"
+    "MX": [
+      "mx.zoho.in",
+      "mx2.zoho.in",
+      "mx3.zoho.in"
+    ]
+    "TXT": "v=spf1 include:zoho.in ~all"
```
Comment (CuteDog5695) [inline domains/_github-pages-challenge-MrPop-rookie.snbhowmik.json:16]: You cannot use CNAME with other records

File: domains/_github-pages-challenge-MrPop-rookie.snbhowmik.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "repo": "https://github.com/MrPop-rookie/MrPop-rookie.github.io",
+  "owner": {
+    "username": "MrPop-rookie",
+    "email": "snbhowmikac@gmail.com",
+    "linkedin": "https://linkedin.com/in/snbhowmik"
+  }
+  "record": {
+    "CNAME": "MrPop-rookie.github.io",
```
Comment (CuteDog5695) [inline domains/_github-pages-challenge-MrPop-rookie.snbhowmik.json:9]: Please add more stuff to your website, as it just has "Hello World" text

File: domains/_github-pages-challenge-MrPop-rookie.snbhowmik.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    "TXT": "9a4944b6a34448f1edc9f4079db605"
```
Comment (CuteDog5695) [inline domains/_github-pages-challenge-MrPop-rookie.snbhowmik.json:10]: You only need TXT for Github Pages verification, go to https://www.is-a.dev/docs/github-pages/ and https://www.is-a.dev/docs/github-pages-verification/ to see what you need to do

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**